JavaShared/src/main/java/org/openzen/zenscript/javashared/prepare/JavaPrepareClassMethodVisitor.java

```diff
@@ -266,19 +266,33 @@ private void visitFunctional(FunctionalMember member, FunctionHeader header, Str
 					signature);
 		} else if (method == null) {
 			if (member instanceof ConstructorMember) {
-				final String signature = context.getMethodSignatureConstructor(member);
+                if(member.isImplicit()) {
+					final String signature = context.getMethodSignature(header, true);
+					method = new JavaCompilingMethod(class_.compiled, new JavaNativeMethod(
+							class_.compiled,
+							getKind(member),
+							"of",
```

Comment:
💭 Question:  
What name do we want to use here for the generated name of implicit ctors?
We probably shouldn't use `<init>` which normal ctors do, and I didn't know of a good name, so I just used `.of` for now. Has anyone an idea what might be a better name?

Reply:
$implicitConstructor?
implicit-constructor?